lib/Elastica/Filter/Bool.php

```diff
@@ -1,6 +1,8 @@
 <?php
 namespace Elastica\Filter;
 
+trigger_error('Use BoolFilter instead. From PHP7 bool is reserved word and this class will be removed in Elastica 4.0', E_USER_DEPRECATED);
```

Comment:
Nice one. The only concern I have is that I don't know what people are using to catch errors. But Lets keep it in.